src/api/app/services/rpmlint_log_extractor.rb

```diff
@@ -0,0 +1,56 @@
+class RpmlintLogExtractor
+  attr_reader :project, :package, :repository, :architecture, :log_content
+
+  def initialize(params)
```

Comment:
Better use regular arguments or positional arguments, instead of hashes

Reply:
I renamed `params` to `attributes`, and keep using it as a hash parameter. We use this convention in other models and services:
```
✗ grep -r -E "def.+initialize.+attributes" src/api/app/{models,services}
src/api/app/models/workflow.rb:  def initialize(attributes = {})
src/api/app/models/workflow/step.rb:  def initialize(attributes = {})
src/api/app/models/scm_webhook.rb:  def initialize(attributes = {})
src/api/app/models/package_service_error_file.rb:  def initialize(attributes = {})
src/api/app/models/project_meta_file.rb:  def initialize(attributes = {})
src/api/app/models/local_build_result/for_package.rb:    def initialize(attributes = {})
src/api/app/models/project_service_file.rb:  def initialize(attributes = {})
src/api/app/models/patchinfo.rb:  def initialize(attributes = {})
src/api/app/models/project_config_file.rb:  def initialize(attributes = {})
src/api/app/models/local_build_statistic/for_package.rb:    def initialize(attributes = {})
src/api/app/models/package_meta_file.rb:  def initialize(attributes = {})
src/api/app/services/rpmlint_log_extractor.rb:  def initialize(attributes = {})
```

Would you say this is a bad practice?

In that case, what kind of arguments should we use in a method when we call it with a permitted slice of `params`?

Reply:
I created another commit _"Use positional arguments"_ that refactors a before_action method and uses positional arguments.

We can merge this commit with the previous one. Or remove it, depending on what we decide.

Reply:
After talking with @krauselukas, we agreed on using named arguments. The `rpmlint_log_params` method still checks for required parameters and return an `ActionController::Parameters` instance later on.